CMakeLists.txt

```diff
@@ -26,6 +26,20 @@ cmake_minimum_required( VERSION 3.0 )
 # header. And don't set LANGUAGES as we are multi-language and header
 # only, so it's irrelevant.
 project( lyra )
+set(PROJECT_VERSION "1.5.1")
+set(PROJECT_DESCRIPTION "A simple to use, composing, header only, command line arguments parser for C++ 11 and beyond.")
+set(PROJECT_HOMEPAGE_URL "https://github.com/bfgroup/Lyra")
+set(CPACK_PACKAGE_NAME "${PROJECT_NAME}")
+set(CPACK_PACKAGE_VERSION "${PROJECT_VERSION}")
+set(CPACK_PACKAGE_VENDOR "${PROJECT_NAME} developers")
+set(CPACK_PACKAGE_DESCRIPTION "${PROJECT_DESCRIPTION}")
+set(CPACK_DEBIAN_PACKAGE_NAME "lib${CPACK_PACKAGE_NAME}-dev")
+set(CPACK_RPM_PACKAGE_NAME "lib${CPACK_PACKAGE_NAME}-devel")
```

Comment:
Are you sure that the CPack command won't fail because it will try to generate packages for every single distribution listed here? May be make sure that the `cpack` won't fail on different platforms.